### PR TITLE
UCT/IB/MLX5: Move EXT atomics to accel TLs

### PR DESCRIPTION
**Pull request by Artemy-Mellanox**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9729/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9726/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9730/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9731/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9762/ for details (Mellanox internal link).

File: src/uct/ib/rc/verbs/rc_verbs_ep.c [hunk 1]
```diff
@@ -398,76 +326,18 @@ ucs_status_t uct_rc_verbs_ep_atomic64_post(uct_ep_h tl_ep, unsigned opcode, uint
...
     }
 
-    return uct_rc_verbs_ep_ext_atomic(ucs_derived_of(tl_ep, uct_rc_verbs_ep_t), op,
-                                      result, sizeof(uint32_t), 0, add, swap,
-                                      remote_addr, rkey, comp);
-#else
-    return UCS_ERR_UNSUPPORTED;
-#endif
+    return uct_rc_verbs_ep_atomic(ucs_derived_of(tl_ep, uct_rc_verbs_ep_t),
+            IBV_WR_ATOMIC_FETCH_AND_ADD, result, value, 0,
```
Comment (yosefe) [inline src/uct/ib/rc/verbs/rc_verbs_ep.c]: indentation

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9789/ for details (Mellanox internal link).

**Comment by Artemy-Mellanox** [comment]
bot:bgate:retest

**Comment by yosefe** [comment]
bot:bgate:retest

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9827/ for details (Mellanox internal link).

**Comment by Artemy-Mellanox** [comment]
bot:bgate:retest

**Pull request merged**